via.js

```diff
@@ -486,12 +486,6 @@ define(['./utils'], function (utils) {
             if (!(self.IC32 & 8) && !self.keys[self.keycol][keyrow]) {
                 self.sdbval &= 0x7f;
             }
-            if (!isMaster && !(self.IC32 & 4)) {
```

Comment:
Anything that removes suspicious code I copied from other emulators without understanding...